test/test.js

```diff
@@ -3,7 +3,7 @@ import test from 'ava';
 import 'babel-register';
 import 'babel-polyfill';
 
-import AnkiExport, { SEPARATOR } from '../src/index';
+import ankiExport, { SEPARATOR } from '../src/index';
```

Comment:
Revert all the changes in this file. 

I see it was a little ambiguous as using `new` with exported function from `index.js` is unnecessary, but now it's just renaming variables and adding whitespaces which should be a separate concern